src/Microsoft.AspNet.Mvc.ModelBinding/Binders/CompositeModelBinder.cs

```diff
@@ -129,12 +132,6 @@ private static bool IsBindingAtRootOfObjectGraph(ModelBindingContext bindingCont
                 PropertyFilter = oldBindingContext.PropertyFilter,
             };
 
-            // validation is expensive to create, so copy it over if we can
```

Comment:
> I don't think it was solving anything
As long we're certain this is certainly the case I'm ok removing it. 

> What security concern do you think can happen here?
Primarily about doing more 2n work (or more?) for a n sized payload.